.github/workflows/osx_universal.yml

```diff
@@ -0,0 +1,28 @@
+name: Universal macOS binary
+on:
+  pull_request:
+    branches:
+      - 'master'
+      - 'release-[0-9]+.[0-9]+'
+    paths:
+      - '**.cmake'
+      - '**/CMakeLists.txt'
+      - '**/CMakePresets.json'
+      - 'cmake.*/**'
+      - '.github/scripts/build_universal_osx.sh'
+      - '.github/workflow/osx_universal_yml'
+
+jobs:
+  universal-binary:
+    runs-on: macos-latest
```

Comment:
Bikeshed: went with `macos-latest` as I didn't want to continually update this, but if we *really* want to replicate the release job it should be `macos-11`. Don't think this should make much difference, but input from maccers are welcome.

Of course none of this would have been a problem to begin with if github just introduced a damn `macos-oldtest` tag *shakes fist*

Reply:
Instead of osx/darwin/apple/macOS/MacOS you mean?

Reply:
yes :)

Reply:
Sure. I wouldn't mind it myself. Renaming the filenames from "osx" to "macos"